doc/user/examples/triple.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import katsdpsigproc.accel
+
+SOURCE = """
+__kernel void multiply(__global float *data, float scale) {
+    data[get_global_id(0)] *= scale;
+}
+"""
+
+ctx = katsdpsigproc.accel.create_some_context(device_filter=lambda x: not x.is_cuda)
+queue = ctx.create_command_queue()
+buf = katsdpsigproc.accel.DeviceArray(ctx, (50,), np.float32, (64,))
+host = buf.empty_like()
+host[:] = np.random.uniform(size=host.shape)
+buf.set(queue, host)
+
+program = ctx.compile(SOURCE)
```

Comment:
It's a bug I noticed when I ran that example. It should be fixed on the user-guide branch.

Reply:
I run the script as it appears in: https://github.com/ska-sa/katsdpsigproc/blob/user-guide/doc/user/examples/triple.py. I still get this error. It might be an issue with the server we are testing on. The Alveo U280 appears on our list of OpenCL devices which may be breaking things.

If it is working your side then I am happy to resolve this issue and just diagnose the problems on our lab machine at some future date.

Reply:
The bug isn't in the script, it's in katsdpsigproc itself. So you need to install katsdpsigproc from the branch for it to work.